### ISSUE TITLE
Crashes when clicking "Select widget"

### ISSUE THREAD

**Issue by CrazyPingu** [KieronQuinn/PixelLauncherMods#30]
When I open the menu to replace the at a glance / the Google search bar and I click "select widget" it crashes. I'm currently on evolution X 6.2.2.

**Comment by KieronQuinn** [comment]
Please send a log.

**Comment by CrazyPingu** [comment]
https://paste.evolution-x.org/05umyS

**Comment by KieronQuinn** [comment]
```
time: 1650300400195
msg: java.lang.NullPointerException
stacktrace: java.lang.NullPointerException
	at com.kieronquinn.app.pixellaunchermods.ui.screens.tweaks.widgetreplacement.widgetpicker.WidgetReplacementPickerViewModelImpl$appWidgets$1.invokeSuspend(WidgetReplacementPickerViewModel.kt:118)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7866)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```

**Comment by KieronQuinn** [comment]
Fixed in 2.2. This one is a bit weird, it appears you have a widget without an app associated with it, if it persists send the log again as it might've simply moved the issue elsewhere.
